source/stylesheets/card.scss

```diff
@@ -1,35 +1,36 @@
 .card {
-	z-index: -1000; 
+  z-index: -1000;
 }
 
 .card {
-	background-color: #F0AA41;
-	width: 28vw;
-	padding: 1rem;
-	text-align: center;
-	border-radius: 20px;
-	height:40vw;
-	margin: 0 auto;
-	margin-top: 10vh;
-	box-shadow: 10px 10px 0.5px 0.5px #ed3e44;;
+  background-color: $yellow;
+  width: $card-width;
+  padding: 1em;
+  text-align: center;
+  border-radius: $base-border-radius;
+  height: $card-height;
+  margin: 0 auto;
+  margin-top: 10vh;
+  box-shadow: $bkg-card-fill;
 }
 
 .card__category {
-	box-sizing: border-box;
-	background-color: #4C95C7;
-	width: 100%;
-	text-align: center;
-	color: white;
-	border-radius: 20px;
-	padding: 1rem;
-	margin: 0 auto;
-	border: 1px solid #65FFFF;
-	font-size: 1.5em;
-	font-weight: bold;
+  background-color: $blue-fill;
+  border-radius: $base-border-radius;
+  border: $base-border;
+  box-sizing: border-box;
```

Comment:
A small aside since this is kind of outside the scope of this PR, I would advise applying that box-sizing everywhere. See https://css-tricks.com/box-sizing/ for an example, the "Universal Box Sizing with Inheritance" example is a great one.